src/components/tests/AreaChart.spec.tsx

```diff
@@ -14,4 +14,40 @@ describe("AreaChart", () => {
         );
         expect(wrapper.find(AfmAreaChart)).toHaveLength(1);
     });
+
+    describe("Stacking", () => {
+        const config = { stackMeasures: true, stackMeasuresToPercent: true };
+
+        it("should reset stackMeasuresToPercent in case of one measure", () => {
+            const wrapper = shallow(
+                <AreaChart
```

Comment:
you can create some component factory parametrized only by Measure and use it in both test cases

Reply:
updated